devtools/moz.build

```diff
@@ -15,6 +15,7 @@ if CONFIG["MOZ_DEVTOOLS"] == "all":
 # `platform` contains all native components
 DIRS += [
     "platform",
+    "replay",
```

Comment:
I wonder if we should put all this in `toolkit/recordreplay/` with the CPP code. It doesn't feel like it belongs in `devtools` really.

Reply:
This is moot (for now at least) since I've moved the code back but a good question for #396